src/packet_writer.cpp

```diff
@@ -71,11 +72,11 @@ void PacketWriter::write(Packet& packet) {
 
 void PacketWriter::write(PDU& pdu, const struct timeval& tv) {
     PDU::serialization_type buffer = pdu.serialize();
-    struct pcap_pkthdr header = { 
```

Comment:
Ah, that's true. Using that syntax is supposed to zero out the rest of the struct AFAIK, so I wonder why it's complaining. Anyway, that works as well, so it's fine.